tests/test_handle_short_events.py

```diff
@@ -0,0 +1,164 @@
+import random
+import unittest
+
+from scripts.cat.cats import Cat
+from scripts.cat.pelts import Pelt
+from scripts.events_module.handle_short_events import HandleShortEvents
+
+
+class TestHandleEvent(unittest.TestCase):
+    pass
+
+
+class TestHandleNewCats(unittest.TestCase):
+    pass
+
+
+class TestHandleAccessories(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Mock classes
+        cls.event_class = type('EventClass', (object,), {})
+
+    def setUp(self):
+        self.test = HandleShortEvents()
+        self.test.chosen_event = self.event_class()
+        self.test.main_cat = Cat()
+
+    def test_misc_appended_to_types(self):
+        self.test.types = []
+
+        self.test.handle_accessories()
+        self.assertIn("misc", self.test.types)
+
+    def test_cat_gets_test_accessory(self):
+        setattr(self.test.chosen_event, "new_accessory", ["TEST"])
+
+        self.test.handle_accessories()
+        self.assertIs(self.test.main_cat.pelt.accessory, "TEST")
+
+    def test_cat_gets_random_wild_accessory(self):
+        setattr(self.test.chosen_event, "new_accessory", ["WILD"])
+
+        self.test.handle_accessories()
+        self.assertIn(self.test.main_cat.pelt.accessory, Pelt.wild_accessories)
+
+    def test_cat_gets_random_plant_accessory(self):
+        setattr(self.test.chosen_event, "new_accessory", ["PLANT"])
+
+        self.test.handle_accessories()
+        self.assertIn(self.test.main_cat.pelt.accessory, Pelt.plant_accessories)
+
+    def test_cat_gets_random_collar_accessory(self):
+        setattr(self.test.chosen_event, "new_accessory", ["COLLAR"])
+
+        self.test.handle_accessories()
+        self.assertIn(self.test.main_cat.pelt.accessory, Pelt.collars)
+
+    def test_notail_cats_do_not_get_tail_accessories(self):
+        setattr(self.test.chosen_event, "new_accessory", [random.choice(Pelt.tail_accessories)])
+        self.test.main_cat.pelt.scars = "NOTAIL"
+
+        self.test.handle_accessories()
+        self.assertNotIn(self.test.main_cat.pelt.accessory, Pelt.tail_accessories)
+
+    def test_halftail_cats_do_not_get_tail_accessories(self):
+        setattr(self.test.chosen_event, "new_accessory", [random.choice(Pelt.tail_accessories)])
```

Comment:
As above - try not to use randomisation in unit tests.

Reply:
As above - Fixed